package.json

```diff
@@ -21,6 +21,7 @@
     "test": "BABEL_ENV=commonjs TS_NODE_FILES=true TS_NODE_PROJECT=tsconfig.commonjs.json NODE_ENV=production mocha --opts mocha.opts src/**/*.test.ts"
   },
   "devDependencies": {
+    "@aics/simularium-viewer": "^2.7.2",
```

Comment:
I think this should be a dependency still, and not a devDependency.   devDependencies are more for things like build tools, transpiler, linter etc